README.md

```diff
@@ -27,8 +37,290 @@ pip install -e .
 
 ## What can AMICAL do for you ?
 
-See [example_NIRISS.py](example_NIRISS.py).
+AMICAL has been developed to provide an easy-to-use solution to process
+**A**perture **M**asking **I**nterferometry (AMI) data from major existing
+facilities:
+[NIRISS](https://jwst-docs.stsci.edu/near-infrared-imager-and-slitless-spectrograph)
+on the JWST (first scientific interferometer operating in space),
+[SPHERE](https://www.eso.org/sci/facilities/paranal/instruments/sphere.html) and
+[VISIR](https://www.eso.org/sci/facilities/paranal/instruments/visir.html) from
+the European Very Large Telescope (VLT) and
+[VAMPIRES](https://www.naoj.org/Projects/SCEXAO/scexaoWEB/030openuse.web/040vampires.web/indexm.html)
+from SUBARU telescope (and more to come).
+
+We focused our efforts to propose a user-friendly interface, though different
+sub-classes allowing to (1) **Clean** the reduced datacube from the standard
+instrument pipelines, (2) **Extract** the interferometrical quantities
+(visibilities and closure phases) using a Fourier sampling approach and (3)
+**Calibrate** those quantities to remove the instrumental biases.
+
+Looking for a quickstart into AMICAL? Just have a look to the example scripts
+made for [NIRISS](example_NIRISS.py) and [SPHERE](example_NIRISS.py).
+
+In addition (4), we include two external packages called
+[CANDID](https://github.com/amerand/CANDID) and
+[Pymask](https://github.com/AnthonyCheetham/pymask) to **analyse** the final
```

Comment:
packaging issue: these _vendored_ libraries should appear as such in the directory tree. Use a `vendor` directory and put them there.

legal issue: you need to copy their respective LICENCES. I checked and saw that CANDID doesn't have one, so it's not actually clear you are _allowed_ to vendor it. Please ask its maintainers for permission at the very least, and lookup how you should acknowledge that permission properly in AMICAL